test/runnable/extra-files/cpp_abi_tests.cpp

```diff
@@ -11,6 +11,8 @@ bool               passthrough(bool                value)     { return value; }
 signed char        passthrough(signed char         value)     { return value; }
 unsigned char      passthrough(unsigned char       value)     { return value; }
 char               passthrough(char                value)     { return value; }
+char16_t           passthrough(char16_t            value)     { return value; }
```

Comment:
I think you're going to have to wrap these in `#if ! __DMC__`, win32 fails with duplicate definition. Ditto for below.